### PR TITLE
Update k_means_clust.py

### PR DESCRIPTION
**Pull request by thor-harsh**
### Describe your change:

* [ ] Add an algorithm?
* [] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [] All new Python files are placed inside an existing directory.
* [] All filenames are in all lowercase characters with no spaces or dashes.
* [] All functions and variable names follow Python naming conventions.
* [] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* []  All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by Bazifrasool** [comment]
Almost there ! just split the long line in two !

File: machine_learning/k_means_clust.py [hunk 1]
```diff
@@ -31,8 +31,8 @@
             record_heterogeneity=heterogeneity,
             verbose=True # whether to print logs in console or not.(default=False)
             )
-  4. Plot the loss function, hetrogeneity values for every iteration saved in
-     hetrogeneity list.
+  4. Plot the loss function, and heterogeneity values for every iteration saved in
```
Comment (tianyizheng02) [inline machine_learning/k_means_clust.py:34]:
```suggestion
  4. Plot the loss function and heterogeneity values for every iteration saved in
```

File: machine_learning/k_means_clust.py [hunk 2]
```diff
@@ -198,10 +198,10 @@ def report_generator(
...
-    Function generates easy-erading clustering report. It takes 2 arguments as an input:
-        DataFrame - dataframe with predicted cluester column;
+    Function generates an easy-erading clustering report. It takes 2 arguments as an input:
+        DataFrame - dataframe with predicted cluster column;
         FillMissingReport - dictionary of rules how we are going to fill missing
-        values of for final report generate (not included in modeling);
+        values of for final report generated (not included in modelling);
     in order to run the function following libraries must be imported:
         import pandas as pd
         import numpy as np
```
Comment (tianyizheng02) [inline machine_learning/k_means_clust.py:207]:
```suggestion
```
This file already imports these libraries so there's no need to specify here

File: machine_learning/k_means_clust.py [hunk 3]
```diff
@@ -198,10 +198,10 @@ def report_generator(
     df: pd.DataFrame, clustering_variables: np.ndarray, fill_missing_report=None
 ) -> pd.DataFrame:
     """
-    Function generates easy-erading clustering report. It takes 2 arguments as an input:
-        DataFrame - dataframe with predicted cluester column;
+    Function generates an easy-erading clustering report. It takes 2 arguments as an input:
+        DataFrame - dataframe with predicted cluster column;
```
Comment (tianyizheng02) [inline machine_learning/k_means_clust.py:202]:
```suggestion
        df - dataframe with predicted cluster column
```

File: machine_learning/k_means_clust.py [hunk 4]
```diff
@@ -198,10 +198,10 @@ def report_generator(
...
         FillMissingReport - dictionary of rules how we are going to fill missing
```
Comment (tianyizheng02) [inline machine_learning/k_means_clust.py:203]:
```suggestion
        fill_missing_report - dictionary of rules on how we are going to fill in missing
```

File: machine_learning/k_means_clust.py [hunk 5]
```diff
@@ -198,10 +198,10 @@ def report_generator(
...
-        values of for final report generate (not included in modeling);
+        values of for final report generated (not included in modelling);
```
Comment (tianyizheng02) [inline machine_learning/k_means_clust.py:204]:
```suggestion
        values for final generated report (not included in modelling);
```

File: machine_learning/k_means_clust.py [hunk 6]
```diff
@@ -309,7 +309,7 @@ def report_generator(
     )  # drop count values except cluster size
```
Comment (tianyizheng02) [inline machine_learning/k_means_clust.py:309]:
```suggestion
    )  # drop count values except for cluster size
```

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

File: machine_learning/k_means_clust.py (hunk 1 above)
Comment (ronnie1612) [reply machine_learning/k_means_clust.py:34]: plt.figure()
plt.plot(loss_list, label='Loss')
plt.xlabel('Iteration')
plt.ylabel('Loss')
plt.title('Loss Function')
plt.legend()
plt.show()
you can try these

**Review by tianyizheng02 — APPROVED** [review]

**Pull request merged**